hamocc/hamocc_init.F90

```diff
@@ -48,6 +48,7 @@ subroutine hamocc_init(read_rest,rstfnm_hamocc)
        &                    dtb,dtbgc,io_stdo_bgc,ldtbgc,                       &
        &                    ldtrunbgc,ndtdaybgc,with_dmsph,l_3Dvarsedpor
   use mo_param1_bgc,  only: ks,init_por2octra_mapping
+  use mo_parambgc_ini,only: ini_parambgc
```

Comment:
Since we plan to have all parameters defined within the module (in the longer run), the name `mo_parambgc_ini` is not really the best, in my opinion. `mo_param_bgc` would be better (but that is too close to `mo_param1_bgc`?). Not sure what to do about it.